staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2657,6 +2662,7 @@ const (
 // The requirements of them are ANDed.
 // It provides a subset of functionality as NodeSelectorTerm.
 // This is an alpha feature and may change in the future.
+// +structType=atomic
 type TopologySelectorTerm struct {
```

Comment:
most uses of this are in a list... should those lists be marked listType=atomic or is that the default?

Reply:
Good catch, those lists need to be marked as atomic. I'll fix.

Reply:
Actually, it should be the default. Verifying.

Reply:
From the docs: "atomic and set apply to lists with scalar elements only" (https://kubernetes.io/docs/reference/using-api/server-side-apply/). 

@apelisse do you know if this is accurate?

Reply:
@jpbetz: Discussed this with @apelisse today, lists are atomic by default and we shouldn't need to do anything extra for them. The documentation is around set lists which aren't applicable in our case (I'll make a todo to update our docs)